src/action.js

```diff
@@ -0,0 +1,21 @@
+
+export const addShirt= ()=>(
+	{type:'ADD_SHIRT',shirts:'shirt',shirtc:1}
+)
+
+
+
+export const addShoe= ()=>(
```

Comment:
you don't necessarily need to send the 1 as data. You can have it automatically update the store by 1.